modules/admin_manual/pages/release_notes.adoc

```diff
@@ -19,6 +20,108 @@
 * xref:changes-in-8-0[Changes in 8.0]
 * xref:changes-in-7-0[Changes in 7.0]
 
+== Changes in 10.2.0
+
+Dear ownCloud administrator, please find below the changes and known issues in ownCloud Server 10.2 that need your attention. You can also read https://owncloud.org/changelog/server/[the full ownCloud Server changelog] for further details on what has changed.
+
+=== Migrations
+
+Please note that this minor release contains database migrations which have an impact on the upgrade duration:
+
+- The table `oc_share` receives a new column (impact depends on the number of shares)
+- The `oc_authtoken` table's login name column size has been increased (impact depends on the number of recently logged in users plus number of app passwords)
+
+=== PHP 5.6/7.0 Deprecation
+
+Following the https://doc.owncloud.com/server/admin_manual/release_notes.html#php-5-6-deprecation[PHP 5.6/7.0 deprecation note with the release of ownCloud Server 10.0.8], ownCloud Server 10.2 *does not support PHP 5.6* anymore. If you're still running on 5.6 it is necessary to upgrade to PHP 7 before upgrading to ownCloud Server 10.2. See the xref:installation/system_requirements.adoc#officially-recommended-supported-options[system requirements in the ownCloud documentation]. Also some new app versions do not support older PHP versions anymore. These will not upgrade if your PHP version is not compatible.
+
+To provide additional time for upgrades, version 10.2 still supports PHP 7.0 as some major Linux distributions continue to deliver support. To ensure secure and performant operations, support for PHP 7.0 will also be discontinued within the next minor versions of ownCloud 10. Please plan an upgrade soon.
```

Comment:
I guess dropping PHP 7.0 may not happen for `10.3` (it might be `10.4` or... depending on other things available for release and how they impact semver - e.g. if a release needs to be called 10.2.1 or 10.3.0)

Maybe change "within the next minor versions of ownCloud 10" to "in an upcoming minor version of ownCloud 10"

If we know an actual timeframe, it would be nice to say it. But maybe we are not yet exactly sure when "some major Linux distributions" will stop their support.